### ISSUE TITLE
Search terms in transcripts should appear in the third line and not the first line when highlighted

### ISSUE THREAD

**Issue by singlesoliloquy** [WGBH-MLA/AAPB2#2283]
## **Is your feature request related to a problem? Please describe.**
Alan requested this and is having trouble searching.

## **Done When**
Search terms in transcripts appear in the third line and not the first line when highlighted

## **Additional context**
![Snap31.png](https://images.zenhubusercontent.com/5e5e7f2e572e74e6537e269f/299f88f2-63f2-493b-a9f8-6b2dd4e95975)

**Comment by singlesoliloquy** [comment]
From Alan:

’ve asked before whether you all can make search terms that appear in a few lines of the transcripts excerpts on the third line of the excerpt rather than the first line. I think you did this for some of the collection, but not for the PBS NewsHour collection. Alyssa is doing a special collection on interviews with foreign leaders, and I found that from 1983 on, the NewsHour consistently used the term “Newsmaker Interview” for such interviews. Searching the collection for that phrase, I found 1,200 hits, but most are not interviews with foreign leaders. It’s sometimes difficult to see in the excerpt who the interview is with (see attached hypersnap).
 
It’s probably not possible, but I’m wondering since this was supposed to have been implemented long ago, whether your developer could quickly implement it now so that the search is not so onerous.
